stdlib/2/email/iterators.pyi

```diff
@@ -0,0 +1,3 @@
+def walk(self) -> None: ...
```

Comment:
This is a generator, it returns an Iterator of something or other (Message objects?).